doc/resources/jenkins/Jenkinsfile-acr-with-git

```diff
@@ -0,0 +1,33 @@
+node {
+   stage('build') {
+    acrQuickBuild azureCredentialsId: env.AZURE_CRED_ID,
+                  resourceGroupName: env.RES_GROUP,
+                  registryName: env.ACR_NAME,
+                  platform: "Linux",
+                  dockerfile: 'doc/resources/jenkins/DockerfileWithMaven',
+                  gitBranch: '',
+                  gitPath: '',
+                  gitRepo: 'https://github.com/Microsoft/todo-app-java-on-azure',
+                  imageNames: [[image: "$env.ACR_REGISTRY/$env.IMAGE_NAME:$env.BUILD_NUMBER"]],
+                  sourceType: 'git',
+                  buildArgs: [[key: 'dbUrl', secrecy: false, value: env.DOCUMENTDB_URI],
+                              [key: 'dbKey', secrecy: true, value: env.DOCUMENTDB_KEY],
+                              [key: 'dbName', secrecy: false, value: env.DOCUMENTDB_DBNAME]]
+  }
+ stage('deploy') {
```

Comment:
Same. Can  you add a stage ('test') to indicate testing should be adding... can be in comments